controllers/ai.controller.js

```diff
@@ -10,50 +11,83 @@ const url = `mongodb+srv://${process.env.DB_USER}:${process.env.DB_PASS}@${proce
 
```

Comment:
Identation seems to be off. Have you downloaded the editorconfig plugin in your editor?